HUD/SetBlipDisplay.md

```diff
@@ -8,23 +8,25 @@ ns: HUD
 void SET_BLIP_DISPLAY(Blip blip, int displayId);
 ```
 
-**displayId Behaviour** <br>
-0 = Doesn't show up, ever, anywhere. <br>
-1 = Doesn't show up, ever, anywhere. <br>
-2 = Shows on both main map and minimap. (Selectable on map) <br>
-3 = Shows on main map only. (Selectable on map) <br>
-4 = Shows on main map only. (Selectable on map) <br>
-5 = Shows on minimap only. <br>
-6 = Shows on both main map and minimap. (Selectable on map) <br>
-7 = Doesn't show up, ever, anywhere. <br>
-8 = Shows on both main map and minimap. (Not selectable on map) <br>
-9 = Shows on minimap only. <br>
-10 = Shows on both main map and minimap. (Not selectable on map) <br>
-Anything higher than 10 seems to be exactly the same as 10. <br>
-<br>
+**displayId Behaviour**
+```
```

Comment:
Code blocks are meant specifically for NativeDB imports

Reply:
I was just copying the formatting of [HUD/SetBlipCategory](https://github.com/citizenfx/natives/blob/99f54ca592832e0bd879198997662b4608ee818f/HUD/SetBlipCategory.md)

If codeblocks have a specific purpose, I can adjust to use a standard list or table instead?

Reply:
Righto, gonna push this back to a Draft PR for now as I don't have immediate time to correct this but I'll bring it back once I've made necessary adjustments.